src/i18n/en.ts

```diff
@@ -378,7 +380,9 @@ const en = {
         "term.metadata.hiddenLabels.label": "Search strings",
         "term.metadata.hiddenLabels.placeholder":
             "Enter a new search string and press 'Add'",
-        "term.metadata.hiddenLabels.placeholder.text": "Add",
+        "term.metadata.hiddenLabels.addButton.text": "Add",
+        "term.metadata.hiddenLabels.addButton.title":
+            "Click to add the hidden label value",
```

Comment:
Why not simply "Click to add the search string" (also, the UI speaks about search strings, so it should be consistent)